src/renderer/fonts/caching_shaper.rs

```diff
@@ -150,7 +150,8 @@ impl CachingShaper {
     pub fn font_base_dimensions(&mut self) -> (u64, u64) {
         let (metrics, glyph_advance) = self.info();
         let font_height = (metrics.ascent + metrics.descent + metrics.leading).ceil() as u64;
-        let font_width = (glyph_advance + 0.5).floor() as u64;
+        let font_width =
+            (glyph_advance + (self.options.width - self.options.size + 0.5)).floor() as u64;
```

Comment:
Why minus the font size here? Isn't that size only vertically?

Reply:
so It looks like the glyph_advance is the width derived the font set, I added in the minus so that the width setting is relative to the font size, <fontFamily>:h12:w12 would be equal to the current view but upping the width relative to the fontsize makes the font go wider, less then the font size makes it go tighter. I did this to make VictorMono look a bit better, it is to tight for my liking. 

this was my config:

vim.opt.guifont = "VictorMono Nerd Font:h12:w13.5"

Reply:
Hm, in my opinion the "width" should be in absolute units, and not change when the actual size (which I interpret as height) changes. But fair enough if that works better for you.

Reply:
uhm, yeah, but from what I can see the width is partly calculated of the font size, the user won't know what the width that was calculated, so something relavtive to the font size is the best they to go by to set the width. It'll also stop them from having to set both the width and size when they make font updates.